OKTA-795656 OIN Wizard: new Use Cases field and automatic use case assignment (#5122)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

* You must have either the super admin or the app and org admin [roles](https://help.okta.com/okta_help.htm?type=oie&id=ext-administrators-admin-comparison) assigned to you.
* Use your company domain email as your username for your Okta admin account (submissions from a personal email account aren't reviewed).

> > **Note:** The app admin role enables you to view and edit details in the OIN Wizard. For testing in the OIN Wizard, you must have both the app admin role and the org admin role assigned to you. The super admin role gives you access to all functionality in the OIN Wizard.

Author: ```suggestion
> **Note:** The app admin role enables you to view and edit details in the OIN Wizard. To test in the OIN Wizard, if you don't have the super admin role, then you must have both the app admin and the org admin roles assigned to you.
```
